### DISCUSSION TITLE
Unable to install package in angular 12 project.

### DISCUSSION THREAD

**Discussion by abdulqayyum935** [abacritt/angularx-social-login#409] [Q&A] [answered]
Hi All,
I'm seeing the below message.

Could not resolve dependency:
npm ERR! peer @angular/common@"^9.0.0 || ^10.0.0 || ^11.0.0" from angularx-social-login@3.5.7
npm ERR! node_modules/angularx-social-login
npm ERR! angularx-social-login@"*" from the root project

Here are the other details of my development environment.

Angular CLI: 12.0.1
Node: 16.2.0
Package Manager: npm 7.13.0
OS: win32 ia32

Angular: 12.0.1
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package Version
@angular-devkit/architect 0.1200.1
@angular-devkit/build-angular 12.0.1
@angular-devkit/core 12.0.1
@angular-devkit/schematics 12.0.1
@schematics/angular 12.0.1
rxjs 6.6.7
typescript 4.2.4

**Comment by mbvb1223** [comment, marked as answer]
npm install --legacy-peer-deps
